build-tools/packages/version-tools/src/versions.ts

```diff
@@ -189,3 +189,23 @@ function getVersionsFromStrings(prefix: TagPrefix, tags: string[]): string[] {
 	semver.sort(versions);
 	return versions;
 }
+
+/**
+ * Checks if the provided version is a test version.
+ *
+ * Test versions are generated from test/ branches and are published to the test feed.
+ *
+ * @param version - The version to check
+ * @returns - True if the version string is a test version, otherwise false
+ *
+ * @example
+ * returns true
+ * isTestVersion("0.0.0-260312-test");
+ *
+ * @example
+ * returns false
+ * isTestVersion("2.1.0-260312");
+ */
+export function isTestVersion(version: string): boolean {
+	return version.startsWith("0.0.0") && version.endsWith("test");
+}
```

Comment:
if you leave this in version-tools, I think it should accept a SemVer object in addition to the string. That's more flexible for callers and is more consistent with what we do for isInternalVersion for example.